modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/ImportObjectDefinitions.testcase

```diff
@@ -0,0 +1,200 @@
+@component-name = "portal-batch-planner"
+definition {
+
+	property custom.properties = "feature.flag.COMMERCE-8087=true";
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Import/Export";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+	}
+
+	tearDown {
+		var testLiferayVirtualInstance = PropsUtil.get("test.liferay.virtual.instance");
+
+		BatchPlanner.batchPlannerTearDown();
+
+		ObjectAdmin.deleteAllCustomObjectsViaAPI();
+
+		if (${testLiferayVirtualInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@priority = 4
+	test CanCreateNewObjectDefinitionAndUpdateExistingObjectDefinitionWithImportJSONFile {
+		property portal.acceptance = "true";
+
+		task ("Given a student object with a custom field name") {
+			ObjectDefinitionAPI.createAndPublishObjectDefinition(
+				en_US_label = "Student",
+				en_US_plural_label = "Student",
+				name = "Student",
+				objectDefinitionExternalReferenceCode = "Student",
+				requiredStringFieldName = "name",
+				requiredStringFieldNameExternalReferenceCode = "studentName");
+		}
+
+		task ("When I import the JSON file which contains the student object definition extended by a new field and a new object definition") {
+			ImportExport.importFile(
+				entityType = "ObjectDefinition (v1_0 - Liferay Object Admin REST)",
+				fileName = "student_and_teacher_objectDefinition_import.json");
```

Comment:
Could you help rename the dependencies files to be the same format with the files [in here ](https://github.com/liferay-headless/liferay-portal/pull/1691/commits/31995415dfe60e5e5c24c85a6d59e7ef60b9315e#diff-a66f77c99079e0a1789304319a75e87bbdf618b57297b218d35dc5a0f26c8a70)? this is how Meg help named recently